patroni/ha.py

```diff
@@ -77,6 +77,15 @@ def failover_limitation(self) -> Optional[str]:
             return 'not watchdog capable'
         return None
 
+    @property
+    def failover_priority(self) -> int:
+        """The failover priority of this node based on its tags.
+
+        :returns: the value of :attr:``~_MemberStatus.tags`` as an integer if possible, otherwise `1`.
+        """
+        fp = parse_int(self.tags.get('failover_priority', 1))
+        return fp if fp is not None else 1
```

Comment:
With this one I also have the feeling that we are copying and pasting code. Again I wonder if we really need that, and if there is a better way of handling.